canonical-web-and-design/maas-ui
# refactor(ui): migrate subnet to a slice

## Done

- Replace the subnet actions and reducers with a slice.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- In the settings open the form to add a DHCP snippet. You should be able to get the list of subnets in the select box.

## Fixes

Fixes: canonical-web-and-design/maas-ui#1653
